src/index.html

```diff
@@ -17,16 +17,29 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="wrapper">
+      <form
+        action="#"
+        data-qa="big"
+        class="search-bar search-bar--big">
+        <input
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+          class="search-bar-input search-bar-input--big"
+        />
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form
+        action="#"
+        data-qa="small"
+        class="search-bar search-bar--small">
+        <input
+          type="text"
+          data-qa="hover"
+          placeholder="Try “Los Angeles“"
+          class="search-bar-input search-bar-input--small"/>
+      </form>
+    </div>
```

Comment:
Consider using semantic HTML tags like `section` or `article` instead of `div` to improve SEO and accessibility.